configs/mini_pupper_config/launch/gazebo.launch.py

```diff
@@ -45,11 +46,13 @@ def generate_launch_description():
         "rviz", default_value="false", description="Launch rviz"
     )
     declare_robot_name = DeclareLaunchArgument(
-        "robot_name", default_value="champ", description="Robot name"
+        "robot_name", default_value="mini_pupper", description="Robot name"
     )
     declare_lite = DeclareLaunchArgument(
         "lite", default_value="false", description="Lite"
     )
+
+    # TODO: check whether ros_control_file shall be declared here or in mini_pupper_description.urdf.xacro
```

Comment:
hi @grassjelly do you know the answer, I didn't find anywhere using this ros_control_file

Reply:
I guess we don't need to define it here, as it is defined below.

https://github.com/mangdangroboticsclub/mini_pupper_description/blob/270eb119e8c2943a1f05106eab30fc9036b2c1fb/urdf/mini_pupper_description.urdf.xacro#L1787